source/_patterns/00-atoms/components/info-bar.mustache

```diff
@@ -1,3 +1,14 @@
-<div class="info-bar info-bar--{{type}}">
-  <span class="info-bar__text">{{{text}}}</span>
+<div class="info-bar info-bar--{{type}}" id="{{id}}"
+     {{#dismiss}}
```

Comment:
I see this will reflect in `patterns-php` providing one more attribute to configure rather than a different `type` for this InfoBar

Reply:
The different type is still needed as a differentiator in the css selectors.

Reply:
Any way to constrain the schema to say that if `type` is `dismissible` than `dismiss` is required?

Reply:
I've tried to do this before, but I don't believe that it's possible to set a required property based on the value of another property.